linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/rule.yml

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: fedora
+
+title: 'Configure System Cryptography Policy'
+
+description: |-
+    To configure the system cyptography policy to use ciphers only from the <tt><sub idref="var_system_crypto_policy" /></tt>
+    policy, run the following command:
+    <pre>$ sudo update-crypto-policies --set <sub idref="var_system_crypto_policy" /></pre>
+
+rationale: |-
+    Centralized cryptographic polices simply applying secure ciphers across an operating system and
```

Comment:
s/simply/simplifies/

Reply:
Centralized cryptography policy simplifies ...

Reply:
actually should be `policies simplify`